cEP-0020.md

```diff
@@ -0,0 +1,280 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describe how a newcomer at coala will become a developer on the
+basis of their activities in the community.
+
+For the implementation of this process, we will build a gamification system.
+So, every newcomer at coala have to complete the processes involved in
+the gamification system in order to become a developer.
+
+## Background
+
+### Newcomer Experience
+
+- How to attract more newcomers
+  - How to cope with the increasing amounts of newcomers
+- How to teach and help newcomers to learn faster so
+  we can keep working on coala.
+  - Talk to Docs team
+- How to improve newcomer -> dev/maintainer ratio
+- How to increase amount of feedback we get
+  - How to use feedback to improve coala and processes
+
+## Why Gamification?
+
+As coala tries to be welcoming organization for newcomers, giving them clear
+pathways and as much direct assistance as possible. However, it is already
+known that newcomers face many barriers while attempting to contribute to
+the open source community for the first time. Some barriers they face include
+orientation issues that can potentially demotivate newcomers from placing
+their first contribution. On the other hand, gamification is widely used to
+engage and motivate people to accomplish task and improve their performance.
+Therefore, the goal of this project is to use gamification to orient and
+motivate newcomers to overcome onboarding barriers to contribute and engage
+with coala and its community.
+
+## Process involved in the Gamification system
+
+As the goal of this project is to make use of metrics and gamification to
+motivate and support newcomers to overcome the orientation barriers to
+contribute to the community, to achieve this goal we have selected three
+game design elements to help newcomers: Levels, Points, and Badges. For
+each of these elements, we will have a set of rules to describe the
+operation of the gamified environment. i.e.:
+
+- **Levels** group different tasks, enabling users to unlock specific and
+contextualize set of actions. Levels will be applied to make newcomers
```

Comment:
"... enabling users to unlock specific and contextualize**d** set**s** of actions."

maybe?